src/components/ResourceRows.js

```diff
@@ -167,7 +168,7 @@ function ResourceRows({ row }) {
       <TableRow hover={true} onClick={() => setOpen(!open)}>
         <TableCell className={classes.cell} align="left">{resc[0]}</TableCell>
         <TableCell className={classes.cell} align="left">{resc[1]}</TableCell>
-        <TableCell className={classes.cell} align="left">{resc[4]}</TableCell>
+        <TableCell className={classes.cell} align="left"><div style={{ display: 'flex', alignItems: 'center'}}>{!isLoadingZoneContext && !validServerHosts.has(resc[4]) && <Tooltip title="Resource hostname doesn't match a known server."><Icon><WarningIcon style={{color: 'orange', fontSize: 22 }} /></Icon></Tooltip>}{resc[4]}</div></TableCell>
```

Comment:
generally good to avoid contractions - for any english-as-a-second-language readers...

`doesn't match` -> `does not match`

Reply:
I see, will make the change!